scripts/puppeteer.server.js

```diff
@@ -1,7 +1,81 @@
-import {createServer} from 'http';
+import webexSDKAdapter from './start';
 
-createServer((request, response) => {
-  response.writeHead(200, {'Content-Type': 'text/html'});
-  response.write('Hello World!');
-  response.end();
-}).listen(1234);
+let MEETING_ID = null;
+
+function getMeeting() {
+  webexSDKAdapter.meetingsAdapter.getMeeting(MEETING_ID).subscribe((meeting) => {
+    document.getElementById('remote-audio').srcObject = meeting.remoteAudio;
+    document.getElementById('remote-video').srcObject = meeting.remoteVideo;
+    document.getElementById('local-audio').srcObject = meeting.localAudio;
+    document.getElementById('local-video').srcObject = meeting.localVideo;
+    document.getElementById('meeting-title').innerHTML = meeting.title;
+  });
+}
+
+document.getElementById('connector').addEventListener('click', async (event) => {
+  event.preventDefault();
+  const status = document.getElementById('connection-status');
+
+  try {
+    if (event.target.name === 'connect') {
+      await webexSDKAdapter.connect();
+      status.innerHTML = 'Connected ✅';
+    } else {
+      await webexSDKAdapter.disconnect();
+      status.innerHTML = 'Disconnected ❌';
+    }
+  } catch (error) {
+    // eslint-disable-next-line no-console
+    console.error('Unable to connect/disconnect:', error);
+  }
+});
+
+document.getElementById('dialer').addEventListener('click', async (event) => {
+  event.preventDefault();
+
+  const destination = document.getElementById('invitee').value;
+
+  try {
+    if (event.target.name === 'create-meeting') {
+      webexSDKAdapter.meetingsAdapter.createMeeting(destination).subscribe(({ID}) => {
+        MEETING_ID = ID;
+        getMeeting();
+      });
+    } else {
+      webexSDKAdapter.meetingsAdapter.addLocalMedia(MEETING_ID);
+    }
+  } catch (error) {
+    // eslint-disable-next-line no-console
+    console.error('Unable to dial:', error);
+  }
+});
+
+document.getElementById('actions').addEventListener('click', async (event) => {
+  event.preventDefault();
+
+  try {
+    switch (event.target.name) {
+      case 'mute-audio':
```

Comment:
Does it makes sense to separate `mute-audio` and `unmute-audio`? Since both call the same function, you can't guarantee that each button does what they are labeled to do. For instance, if you click mute and then you click it again,  even though it says mute, it will do an "unmute"